andela/ah-frontend-kronos
# #166879999 Users should be able to rate articles

#### What does this PR do?
Users should be able to rate articles

#### Description of Task to be completed?
A 5-star rating system is the end goal. Let's have authenticated users get visual feedback of the average rating of their articles

#### How should this be manually tested?
Follow the setup steps on the [readme page](https://github.com/andela/ah-frontend-kronos) of this repo to set-up locally, you will notice star ratings on the articles and are able to rate when reading the article.

#### What are the relevant pivotal tracker stories?
[#166879999](https://www.pivotaltracker.com/story/show/166879999)

#### Screenshots (if appropriate)
<img width="732" alt="Screenshot 2019-07-26 at 10 38 45" src="https://user-images.githubusercontent.com/28473847/61934795-cfa00400-af91-11e9-9794-5dd0b2de303e.png">
![Uploading Screenshot 2019-07-26 at 10.39.59.png…]()
